.circleci/config.yml

```diff
@@ -279,6 +299,9 @@ workflows:
             branches:
               only: master
       - docs-update-grakn-dependency-on-dev-branch:
+          filters:
```

Comment:
check if possible to trigger this job before merging rather than after

Reply:
will have a discussion with Haikal and probably create a new PR to address it.